shapeshift/web
# fix: qr code not filling address input

## Description

- Fixes the address input not filling after a user scans a QR code

## Notice


## Pull Request Type


## Issue (if applicable)

closes #2006

## Risk

None, this just changes setting the field value to `SendFormFields.Input` vs. `SendFormFields.Address` previously following #1948, to match the new input to address logic, including validation

## Testing

Tested with ENS, Unstoppable Domains, Trust Wallet and MetaMask mobile (broken but irrelevant to this diff, see https://github.com/shapeshift/web/issues/2059)

- Sends should show no regressions on ETH/ERC20s/BTC and ATOM sends (to an address or to an ENS/Unstoppable domain)

1. Select asset
2. Send
3. Click QR code
4. Scan an address' QR code
5. Voila!

## Screenshots (redacted to zero address)

MetaMask Mobile:

<img width="481" alt="image" src="https://user-images.githubusercontent.com/17035424/174304926-bed4d1c9-1a12-484e-a1d5-62178ac32fc9.png">

Trust Wallet:

<img width="470" alt="image" src="https://user-images.githubusercontent.com/17035424/174305047-8de7b5ae-0c86-4a4e-982c-aad4aae90205.png">

Unstoppable Domains:

<img width="483" alt="image" src="https://user-images.githubusercontent.com/17035424/174305331-fa0f275f-26e9-4aab-b9ea-0ed282c1bc71.png">

ENS:

<img width="483" alt="image" src="https://user-images.githubusercontent.com/17035424/174305984-9d6b1ef8-7520-4549-80ca-4e3219826b50.png">
